systemc/NoSysC_test.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iostream>
+#include <memory>
+#include <vector>
+#include <gtest/gtest.h>
+#include <glog/logging.h>
+#include "model_rsa.h"
+#include "verilog/dtype/vint.h"
+#include "modellib/cport.h"
+#include "modellib/cvalidready.h"
+#include "modellib/cmodule.h"
+
+using namespace verilog;
+using namespace std;
+
+struct TwoPower_NSC : public Module<TwoPower_NSC> {
```

Comment:
Suggest to move the `ExtendKeyType` definition here.
The type has different meaning in Montgomery.